distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3985,7 +3986,7 @@ None
 
 Using the REST Endpoint for example, request a metacard with the transform option set to the HTML shortname.
 ----
-http://localhost:8181/services/catalog/0123456789abcdef0123456789abcdef?transform=html
+${public_url}/services/catalog/0123456789abcdef0123456789abcdef?transform=html
```

Comment:
Definitely clarify, and we should include that note. But perhaps instead of changing the instances of `${public_url}`, instead we just add a simple note? 

```
Public, insecure URLs can be used here as shown. We highly encourage the use of secure URLs. 
```

Thoughts?